### PR TITLE
Added MobilenetV3Small

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

**Review by alexsu52 — APPROVED** [review]
Looks good to me

File: beta/tests/tensorflow/test_models/mobilenet_v3.py [hunk 1]
```diff
@@ -11,208 +11,8 @@
...
 """
 
-from tensorflow.python.keras import backend
-from tensorflow.python.keras import models
-from tensorflow.python.keras.applications import imagenet_utils
-from tensorflow.python.keras import layers
+from beta.examples.tensorflow.common.models import MobileNetV3Small as mobilenet_v3_small
 
 
 def MobileNetV3Small(input_shape=None):
```
Comment (andrey-churkin) [inline beta/tests/tensorflow/test_models/mobilenet_v3.py:17]: I think we can remove this file. @alexsu52, do you agree with me?

Comment (l-bat) [reply beta/tests/tensorflow/test_models/mobilenet_v3.py:17]: Test failed without it

Comment (alexsu52) [reply beta/tests/tensorflow/test_models/mobilenet_v3.py:17]: @andrey-churkin essentially, we can remove this file and import the model from the example directly. But I hope TF will fix the issue with this model and we will add the correct implementation to the test. I would leave this file in the tests.

**Review by alexsu52 — APPROVED** [review]

**Review by andrey-churkin — APPROVED** [review]

**Comment by alexsu52** [comment]
Please, update https://github.com/openvinotoolkit/nncf/blob/develop/README.md too

**Pull request merged**